Pod/Classes/LayoutConvenience.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ConstraintConvenience.swift
+//  Pods
+//
+//  Created by Nick DiStefano on 12/11/15.
+//
+//
+
+import Foundation
+
+public func activateVFL(format: String, options: NSLayoutFormatOptions = [], metrics: [String : AnyObject]? = nil, views: [String : AnyObject]) {
```

Comment:
A general question: This function really just saves one line of code but it adds complexity by adding more grammar. Do we want stuff like this?